tests/test_kwargs_handlers.py

```diff
@@ -48,6 +60,8 @@ def test_kwargs_handler(self):
         assert MockClass(a=2, b=True).to_kwargs() == {"a": 2, "b": True}
         assert MockClass(a=2, c=2.25).to_kwargs() == {"a": 2, "c": 2.25}
 
+    # this test leaking its mixed precision to all next ones (shoudl the state be reset at the end of the test?)
```

Comment:
Yes it should be

Reply:
used `AccelerateTestCase` for accelerator state cleanup. let's see if that works.